.travis.yml

```diff
@@ -11,40 +11,51 @@ matrix:
   include:
     - python: "3.6"
       env: TOXENV=py36
-    - python: "3.7.7"
+    - python: "3.7"
       env: TOXENV=py37
     - python: "3.8"
       env: TOXENV=py38
     - python: "3.9"
       env: TOXENV=py39
-    - python: "3.9-dev"
-      env: TOXENV=py39-dev
+    - python: "3.10"
+      env: TOXENV=py310
     # PyPy Environments
-    - python: "pypy2.7-6.0"
-      env: TOXENV=pypy
-    - python: "pypy3.5-7.0"
-      env: TOXENV=pypy3
+    - python: "pypy3.6-7.3.3"
+      env: TOXENV=pypy36
+    - python: "pypy3.7-7.3.9"
+      env: TOXENV=pypy37
+    - python: "pypy3.8-7.3.9"
+      env: TOXENV=pypy38
+    - python: "pypy3.9-7.3.9"
+      env: TOXENV=pypy39
     # An extra environment where additional packages are not installed
     - python: "3.9"
       env:
         - TOXENV=bare
 
 install:
   - pip install babel
-  # upgrade tox, pip, and virtualenv so Python 3.6 will build crytography:
-  # https://travis-ci.community/t/pip-install-cryptography-fails-on-py36/11233
-  - pip install -U tox pip virtualenv
```

Comment:
Any value in bringing this back? This 1 liner allowed all pip calls to install cryptography (regardless of what version)

Reply:
It's right there, just tweaked it a bit.

```
pip install --upgrade tox pip virtualenv wheel
```